src/components/TopBar/components/SearchField/SearchField.scss

```diff
@@ -99,7 +99,7 @@ $stacking-order: (
 
 .Clear {
   @include recolor-icon(
-    var(--p-interactive-neutral-elevation-4, color('ink', 'lightest'))
+    var(--p-neutral-action-pressed, color('ink', 'lightest'))
```

Comment:
Also seems like the wrong role, maybe a new role for `neutral-icon`, `neutral-icon-hovered`, `neutral-icon-pressed`?

Reply:
I don't think so, since this is in-effect an icon-only button